Description: Relax unicode-joining-type deps
 Relax version
Author: Emmanuel Arias <eamanu@debian.org>
Last-Update: 2025-03-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@
 path = "src/lib.rs"
 
 [dependencies.unicode-joining-type]
-version = "0.7.0"
+version = "1.0.0"
 
 [dev-dependencies.assert_matches]
 version = "1.3"
